rules/dropbox_rules/dropbox_linked_team_application_added.yml

```diff
@@ -0,0 +1,121 @@
+AnalysisType: rule
+Description: An application was linked to your Dropbox Account
+DisplayName: Dropbox Linked Team Application Added
+Enabled: false
+Filename: dropbox_linked_team_application_added.py
+Reference: Ensure that the application is valid and not malicious
+Runbook: Verify that this is expected
+Severity: Low
+Tags:
+    - dropbox
+Tests:
+    - ExpectedResult: true
+      Log:
+        actor:
+            _tag: user
+            user:
+                _tag: team_member
+                account_id: dbid:xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+                display_name: user_name
+                email: user@domain.com
+                team_member_id: dbmid:xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+        context:
+            _tag: team
+        details:
+            .tag: app_link_team_details
+            app_info:
+                .tag: team_linked_app
+                app_id: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+                display_name: dropbox-app-name
+        event_category:
+            _tag: apps
+        event_type:
+            _tag: app_link_team
+            description: Linked app for team
+        involve_non_team_member: false
+        origin:
+            access_method:
+                .tag: api
+                request_id: dbarod:xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+            geo_location:
+                city: Los Angeles
+                country: US
+                ip_address: 1.2.3.4
+                region: California
+        timestamp: "2023-02-16 20:39:34"
+      Name: Test-5f682
```

Comment:
it would be helpful to give the `Name` a value that describes what the test does

Reply:
Addressed in [`1297205` (#687)](https://github.com/panther-labs/panther-analysis/pull/687/commits/1297205451fb38f8c92a23625fda514f8c5bb864).